src/services/healthcareProfessionalService.ts

```diff
@@ -93,6 +96,14 @@ Promise<Result<gqlTypes.HealthcareProfessional[]>> {
             searchRef = searchRef.where('degrees', 'array-contains-any', filters.degrees)
         }
 
+        if (shouldFilterByNames) {
+            // Extra check is needed to get the first index of names array filter.
+            // Could not reuse hasNamesFilter due to TypeScript syntax error.
+            if (filters.names && filters.names?.length > 0) {
```

Comment:
Nit: you also don't need the > 0 here
```suggestion
            if (filters.names && filters.names?.length) {
```